### PR TITLE
Added support for SLES12SP2 repositories

### PR DESCRIPTION
**Pull request by jsuchome**
I have no idea what to enter as `key_md5` values...

Reply from @vuntz at irc: "keeping the same values as for SP1 should be fine anyway, as these are the IBS keys and they're the same between SP1 & SP2 (and Cloud 7 & Cloud 7"

### CODE REVIEW

File: crowbar_framework/config/repos-cloud.yml [hunk 1]
```diff
@@ -135,3 +135,119 @@ suse-12.1:
...
+      name: "SUSE-OpenStack-Cloud-7-Updates"
+      required: "mandatory"
+      features: ["openstack"]
+#      repomd:
+#        tag: "obsrepository://build.suse.de/SUSE:Updates:OpenStack-Cloud:7:x86_64/update"
+#        key_md5: "9a62177e1c6852d48453ed3909956d6b"
+      url:
+      smt_path: "SUSE/Updates/OpenStack-Cloud/7/x86_64/update"
+      ask_on_error: false
+    sle12-sp1-ha-pool:
```
Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: s/sp1/sp2/ ;-)

Comment (vuntz) [inline crowbar_framework/config/repos-cloud.yml]: (same below for a few other entries)

File: crowbar_framework/config/repos-cloud.yml [hunk 2]
```diff
@@ -135,3 +135,119 @@ suse-12.1:
...
+      url:
+      smt_path: "SUSE/Updates/SLE-SERVER/12-SP2/x86_64/update"
+      ask_on_error: false
+    cloud:
+      name: "Cloud"
+      required: "mandatory"
+      features: ["openstack"]
+      repomd:
+        tag: ["obsproduct://build.suse.de/SUSE:SLE-12-SP2:Update:Products:Cloud7/suse-openstack-cloud/7/cd/x86_64",
+              "obsproduct://build.suse.de/Devel:Cloud:7:Mitaka/suse-openstack-cloud/7/cd/x86_64",
```
Comment (aplanas) [inline crowbar_framework/config/repos-cloud.yml]: Not sure to take mitaka here

**Comment by aplanas** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**